docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -566,6 +566,13 @@ Write operations are supported for tables stored on the following systems:
   detects write collisions on ADLS Gen2 when writing from multiple Trino
   clusters, or from as any number of open source Delta Lake clusters.
```

Comment:
```suggestion
  detects write collisions on ADLS Gen2 when writing from multiple Trino
  or Delta Lake clusters.
```
I can't figure out what this is trying to say and I don't think we need to tell people that Delta Lake is open source here, so let's clean it up.

Reply:
I rewrote a bit this statement to be a bit more concise, but still tried to keep its essence.